lib/runtime/wasmer/exports_test.go

```diff
@@ -1210,3 +1217,120 @@ func TestInstance_TransactionPaymentCallApi_QueryCallFeeDetails(t *testing.T) {
 		require.Equal(t, test.expect, details)
 	}
 }
+
+func TestInstance_GrandpaGenerateKeyOwnershipProof(t *testing.T) {
+	t.Parallel()
+	runtime := NewTestInstance(t, runtime.WESTEND_RUNTIME_v0929)
+	identity := common.MustHexToBytes("0x88dc3417d5058ec4b4503e0c12ea1a0a89be200fe98922423d4334014fa6b0ee")
+	identityPubKey, _ := ed25519.NewPublicKey(identity)
+	var authorityID [32]byte
+	copy(authorityID[:], identityPubKey.Encode())
+
+	encodedOpaqueKeyOwnershipProof, err := runtime.GrandpaGenerateKeyOwnershipProof(uint64(0), authorityID)
+	require.NoError(t, err)
+	// Since the input is not valid with respect to the runtime and empty proof is returned
+	require.Empty(t, encodedOpaqueKeyOwnershipProof)
```

Comment:
Should we return an error from `runtime.GrandpaGenerateKeyOwnershipProof` is the proof is empty? 🤔

Reply:
I was duplicating @kishansagathiya's PR so perhaps good to talk about with him as well on if this should be an error or not